### ISSUE TITLE
3.2 vs 3.17 [Request]

### ISSUE THREAD

**Issue by Moxiesan** [skotlex/ffxiv-material-ui#162]
This is in regards to the Discord version.

What I loved most about the Discord Material UI was that all the buttons and icons were flush with the UI. I love a nice clean and simple look to my UI, which it delivered nearly flawlessly. With the updated art, a lot of the buttons and icons now have squares around them, which really interrupts the windows. Examples: class icons in the crafting menu, the history button on the marketboard, etc. Another way more drastic change was going from the glow highlight around ability buttons on the hotbar to the blue border. I much prefer the glow because again, it was really clean.

Long story short, I was hoping we could get that options for the mouseovers to be glows and if you could bring back the flush looking interface again.

A side note; Is there a way to chop off the end of sun indicator around the minimap to where it's just like a little bump rotating around the map? It would help minimize and compact the UI a little more while still being informational and beautiful.

Thank you for your incredible work though! It is the best UI mod I've ever had the pleasure of using!

**Comment by skotlex** [comment]
I wonder if what you're saying is that you want to use the black version of glow icon in the discord version. And I'm sorry but I can't understand what side note says. The performance of the translator is confusing to me. I'm sorry I didn't understand you even though you wrote me a long article. Can you explain it with an image? It doesn't have to be a smooth image

**Comment by Moxiesan** [comment]
![ss](https://user-images.githubusercontent.com/3701531/82155626-43264100-9844-11ea-9e77-a437c454b862.png)

Light blue circle - removing the teardrop part of it and leaving the base line there.
Red circle - in 3.17 this is the glow around the icons in the discord version of UI

Hope that helps clarify.

**Comment by skotlex** [comment]
I'm not sure about the mini map, but I'll think about it. 
For icon frame, export the iconA_Frame item of 3.17 using textool and apply it.

**Comment by skotlex** [comment]
Please use past patches for icon_frame as previously mentioned. And since inquiries about Light Blue Circle, it's an area of taste, I decided not to work on this. I'm sorry I couldn't give you the answer you wanted.
